taskt/Core/Automation/Commands/IE Browser Commands/IEElementActionCommand.cs

```diff
@@ -287,145 +302,6 @@ private bool FindQualifyingElement(EnumerableRowCollection<DataRow> elementSearc
-                                       select rw.Field<string>("Parameter Value")).FirstOrDefault();
-
-                string convertedAttribute = Convert.ToString(element.getAttribute(attributeName));
-
-                convertedAttribute.StoreInUserVariable(sender, variableName);
-            }
-        }
-
-        private int FindElementXPosition(MSHTML.IHTMLElement obj)
-        {
-            int curleft = 0;
-            if (obj.offsetParent != null)
-            {
-                while (obj.offsetParent != null)
-                {
-                    curleft += obj.offsetLeft;
-                    obj = obj.offsetParent;
-                }
-            }
-
-            return curleft;
-        }
-
-        public int FindElementYPosition(MSHTML.IHTMLElement obj)
-        {
-            int curtop = 0;
-            if (obj.offsetParent != null)
-            {
-                while (obj.offsetParent != null)
-                {
-                    curtop += obj.offsetTop;
-                    obj = obj.offsetParent;
-                }
-            }
-
-            return curtop;
-        }
-
-        public override List<Control> Render(frmCommandEditor editor)
-        {
-            base.Render(editor);
-
-            RenderedControls.AddRange(CommandControls.CreateDefaultInputGroupFor("v_InstanceName", this, editor));
-
-            SearchParameterControls = new List<Control>();
-            SearchParameterControls.Add(CommandControls.CreateDefaultLabelFor("v_WebSearchParameter", this));
-            SearchParameterControls.AddRange(CommandControls.CreateUIHelpersFor("v_WebSearchParameter", this, new Control[] { SearchGridViewHelper }, editor));
-
-            SearchParameterControls.Add(SearchGridViewHelper);
-            RenderedControls.AddRange(SearchParameterControls);
-
-            ElementActionDropdown = (ComboBox)CommandControls.CreateDropdownFor("v_WebAction", this);
-            RenderedControls.Add(CommandControls.CreateDefaultLabelFor("v_WebAction", this));
-            RenderedControls.AddRange(CommandControls.CreateUIHelpersFor("v_WebAction", this, new Control[] { ElementActionDropdown }, editor));
-            ElementActionDropdown.SelectionChangeCommitted += ElementActionDropdown_SelectionChangeCommitted;
-            RenderedControls.Add(ElementActionDropdown);
-
-            ElementParameterControls = new List<Control>();
-            ElementParameterControls.Add(CommandControls.CreateDefaultLabelFor("v_WebActionParameterTable", this));
-            ElementParameterControls.AddRange(CommandControls.CreateUIHelpersFor("v_WebActionParameterTable", this, new Control[] { ElementsGridViewHelper }, editor));
-            ElementParameterControls.Add(ElementsGridViewHelper);
-            RenderedControls.AddRange(ElementParameterControls);
-
-            return RenderedControls;
-        }
-
         private void ElementActionDropdown_SelectionChangeCommitted(object sender, EventArgs e)
         {
 
```

Comment:
Line 310 - comparisonComboBox is declared but never used.

Reply:
removed